src/Application/Ignore/IgnoredErrorHelperResult.php

```diff
@@ -0,0 +1,227 @@
+<?php
+
+declare(strict_types=1);
+
+namespace TwigStan\Application\Ignore;
+
+use Symfony\Component\Filesystem\Path;
+use TwigStan\Application\TwigStanError;
+use TwigStan\Twig\SourceLocation;
+
+final class IgnoredErrorHelperResult
+{
+    /**
+     * @param list<string> $errors
+     * @param array<array<mixed>> $otherIgnoreErrors
+     * @param array<string, array<array<mixed>>> $ignoreErrorsByFile
+     * @param (string|mixed[])[] $ignoreErrors
```

Comment:
I'm no fan of these untyped arrays.

I was wondering. Why don't we make IgnoredError a value object, that is the representation of the Ignored Error.

Then this becomes:
```suggestion
     * @param list<IgnoredError> $ignoreErrors
```

Reply:
The benefit is that we can now have type safety on the IgnoredError value object. And move some of the helper methods there too. This might improve maintainability.